week-2/Homework/mandatory/3-playing-computer.js

```diff
@@ -15,28 +15,37 @@
   7. What is the value of the "a" outer variable when "f1" is called for the first time?
 */
 
-let x = 2;
-let a = 6;
+let x = 1;
+let a = 2;
+let b = 3;
+let i = 0;
+let g = 0;
+let j = 0;
 
 const f1 = function(a, b) {
+  g++;
```

Comment:
Ha, this is a clever approach! The objective of this exercise is to make you think about how the code actually works, so please make sure you understand that 🙂